packages/api/models/schemas.js

```diff
@@ -179,14 +179,35 @@ module.exports.granule = {
     createdAt: {
       type: 'number',
       readonly: true
+    },
+    productVolume: {
+      type: 'number',
+      readonly: true
+    },
+    beginningDateTime: {
+      type: 'string',
+      readonly: true
+    },
+    endingDateTime: {
+      type: 'string',
+      readonly: true
+    },
+    timeToArchive: {
+      type: 'number',
+      readonly: true
+    },
+    productionDateTime: {
+      type: 'number',
+      readonly: true
```

Comment:
There are some fields missing, such as timeToPreprocess, timeToXfer.  I think timeToXfer is the time for transfer files from remote provider to staging.  timeToPreprocess is other time spent, such as validating etc.

https://wiki.earthdata.nasa.gov/display/CUMULUS/EMS+Generation+Roadmap

Reply:
On the wiki they are listed as all the same, which is what I went by. This is just for what is being indexed, but all fields would be present in the EMS record.